src/gui/log/loglistview.cpp

```diff
@@ -0,0 +1,140 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * In addition, as a special exception, the copyright holders give permission to
+ * link this program with the OpenSSL project's "OpenSSL" library (or with
+ * modified versions of it that use the same license as the "OpenSSL" library),
+ * and distribute the linked executables. You must obey the GNU General Public
+ * License in all respects for all of the code used other than "OpenSSL".  If you
+ * modify file(s), you may extend this exception to your version of the file(s),
+ * but you are not obligated to do so. If you do not wish to do so, delete this
+ * exception statement from your version.
+ */
+
+#include "loglistview.h"
+
+#include <QApplication>
+#include <QClipboard>
+#include <QFontMetrics>
+#include <QKeyEvent>
+#include <QPainter>
+#include <QStyle>
+#include <QStyledItemDelegate>
+
+#include "logmodel.h"
+#include "uithememanager.h"
+
+namespace
+{
+    const QString SEPARATOR = QStringLiteral(" - ");
+
+    int horizontalAdvance(const QFontMetrics &fontMetrics, const QString &text)
+    {
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 11, 0))
+        return fontMetrics.horizontalAdvance(text);
+#else
+        return fontMetrics.width(text);
+#endif
+    }
+
+    QString logText(const QModelIndex &index)
+    {
+        return QString::fromLatin1("%1%2%3").arg(index.data(BaseLogModel::TimeRole).toString(), SEPARATOR
+                                                 , index.data(BaseLogModel::MessageRole).toString());
+    }
+
+    class LogItemDelegate : public QStyledItemDelegate
+    {
+    public:
+        using QStyledItemDelegate::QStyledItemDelegate;
+
+    private:
+        void paint(QPainter *painter, const QStyleOptionViewItem &option, const QModelIndex &index) const override
+        {
+            painter->save();
+            QStyledItemDelegate::paint(painter, option, index); // paints background, focus rect and selection rect
+
+            const QStyle *style = option.widget ? option.widget->style() : QApplication::style();;
+            const QRect textRect = style->subElementRect(QStyle::SE_ItemViewItemText, &option, option.widget)
+                                   .adjusted(1, 0, 0, 0); // shift 1 to avoid text being too close to focus rect
+
+            QFont font = option.font;
+            if (option.font.pointSize() > 0)
+                font.setPointSize(option.font.pointSize()); // somehow this needs to be set directly else painter will use default font
+            painter->setFont(font);
```

Comment:
This should be equivalent `painter->setFont(option.font);`, could you please test it again?

Reply:
![image](https://user-images.githubusercontent.com/34717789/79306288-9722ca80-7f12-11ea-825b-070a38f6be1a.png)